src/libraries/System.Security.Cryptography/tests/OpenSslNamedKeysTests.manual.cs

```diff
@@ -0,0 +1,194 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Tests
+{
+    // See osslplugins/README.md for instructions on how to build and install the test engine and setup for TPM tests.
+    [SkipOnPlatform(~TestPlatforms.Linux, "Named keys tests are only supported on Linux")]
```

Comment:
These tests can run on macOS, too (and should).  When OpenSSL is completely unavailable they will have... a behavior.  Which is probably good to verify.  Hopefully it's a graceful failure and not a whole process teardown.

When OpenSSL is available, they should work.

Unless we actually blocked macOS in code, in which case the UnsupportedOSAttribute values should also reflect that.